public/app/models/query/Group.js

```diff
@@ -0,0 +1,90 @@
+// Based on http://kindohm.github.io/knockout-query-builder/js/group.js
+(function () {
+    'use strict';
+
+    define(
+        [
+            'lodash',
+            'knockout',
+            'models/query/Condition'
+        ],
+        function (_, ko, Condition) {
+            var Group;
```

Comment:
Declaring this as a variable so that it can be used recursively when a group adds a child group, and in parsing.